docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
 
-* deploys NPSP's five dependent managed packages in the right sequence;
-* delivers unpackaged Record Types for the Account and Opportunity objects;
-* installs NPSP;
-* and performs final configuration to make the customer's experience better, like setting up Global Actions and delivering translations.
+A **package** is a container for something as small as an individual component or as large as a set of related apps. After creating a package, you can distribute it to other Salesforce users and organizations, including those outside your company.
 
-CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating in the delivery of new versions of the application to our users through MetaDeploy.
+**Unmanaged packages** are typically used to distribute open-source (non-proprietary) features or application templates to provide developers with the basic building blocks for an application. Once the components are installed from an unmanaged package into a specific org, it's what's known as an org implementation. These freshly installed components can be edited as seen fit by the owners of this implementation. The developer who created and uploaded the unmanaged package has no control over the installed components, and can't change or upgrade them.
+ 
+**Managed packages** are typically used by Salesforce partners to distribute and sell applications to customers. They are proprietary code that can only be upgraded and deployed by the developer that built them. To ensure seamless upgrades, certain destructive changes, like removing objects or fields, cannot be performed.
 
-Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
+In CumulusCI, packages are also built, stored and deployed via **projects**.
 
 Projects
 --------
 
-When you work with CumulusCI, you do so inside a project. A project is a version control repository that contains Salesforce metadata as well as the CumulusCI automation that builds and releases the project. A project usually has a one-to-one relationship with a managed package, if building a package, or with an org implementation. A project may constitute the entirety of a product in the Product Delivery Model, or may be one of multiple projects making up a product.
+When you work with CumulusCI, you do so inside a project. A project is an individual git repository (think GitHub) that contains Salesforce metadata as well as the CumulusCI automation (e.g., tasks and flows) that builds and releases the project. Most importantly, a project can build one - and only one(!) - package in its repository. This one-to-one relationship between project and package is what allows version control to track any changes your team made to the repository with minimal collision.
```

Comment:
```suggestion
When you work with CumulusCI, you do so inside a project. A project is an individual Git repository (think GitHub) that contains both Salesforce metadata and CumulusCI automation (such as tasks and flows) that builds and releases the project. Most importantly, a project can build one—and only one(!)—package in its repository. This one-to-one relationship between project and package is what allows version control to track any changes your team makes to the repository.
```
I like this explanation! Consider whether you really need the GitHub parenthetical? It seems more distracting than illuminating.

Reply:
@lianeyee I'll remove the "Think GitHub" bit.

@davidmreed IIRC, "one-to-one relationship" came from the original text here. I discussed that in greater detail with @Br4nd0R, and that's where I came up with this description. If I'm not still not describing it accurately, what can we say that stresses the importance of the one-to-one relationship? (@davisagli, you might want to chime in here, too.)

Reply:
I agree that it's inaccurate to say that the 1:1 package-repository relationship is what makes version control possible.  One package per repository is more of a convention that CumulusCI assumes in a lot of places -- it's possible to imagine storing multiple packages in the same repository and setting up custom CumulusCI flows to work with them, but you would be "working against the grain" and not able to take advantage of the default flows that CumulusCI provides. So the main point is that you'll have a better experience with CumulusCI if you accept its premise that each package should go in its own repository.

Reply:
@davisagli Would you advise we just leave the language as is in order to "push" people to stick with the 1:1 relationship?

Reply:
@justinvholt how about:

When you work with CumulusCI, you do so inside a project. A project is an individual Git repository (think GitHub) that contains both Salesforce metadata and CumulusCI automation (such as tasks and flows) that builds and releases the project. If you are building multiple packages, we strongly recommend organizing each package as a separate project in its own repository. CumulusCI's standard automation assumes that there is one package per repository, so it will work the best if you follow this convention.